app/controllers/idt/api/v2/distributions_controller.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+class Idt::Api::V2::DistributionsController < Idt::Api::V1::BaseController
+  protect_from_forgery with: :exception
+  before_action :verify_access
+
+  # rubocop:disable Metrics/MethodLength, Naming/AccessorMethodName, Metrics/CyclomaticComplexity
+  def get_distribution
+    # rubocop:enable Metrics/MethodLength, Naming/AccessorMethodName, Metrics/CyclomaticComplexity
+    distribution_id = params[:distribution_id]
+    # Checks if the distribution id is blank and if it exists with the database
+    if distribution_id.blank? || !valid_id?(distribution_id)
+      render_error(400, "Distribution Does Not Exist Or Id is blank", distribution_id)
+      return
+    end
+
+    begin
+      # Retrieves the distribution package from the PacMan API
+      distribution = PacManService.get_distribution_request(distribution_id)
+      response_code = distribution.code
+
+      if response_code != 200
+        fail StandardError
+      end
+      # Handles errors when making any requests both from Pacman and the DB
+    rescue StandardError
+      case response_code
+      when 400
+        render_error(400, "Participant With UUID Not Valid", distribution_id)
+      when 404
+        render_error(404, "Distribution Does Not Exist At This Time", distribution_id)
```

Comment:
Based on some changes with 21123, distributions will now be created prior to being established in Pacman. 

```suggestion
        render_error(404, "A distribution could not be located given the ID provided.", distribution_id)
```

As a result, I think the language should be tweaked a bit so that the client isn't thinking that it's a WIP.

Reply:
Oh, I just saw the other 404 up top.

```json
{
  "id": 1234,
  "status": "PENDING_ESTABLISHMENT"
}
```

Something like that may be better here then. We have a record of it, but Pacman doesn't!

Reply:
Updated